### ISSUE TITLE
Gekko dies on trading XRP/BTC on bitfinex

### ISSUE THREAD

**Issue by stefano-neri** [askmike/gekko#1724]
**Note: for support questions, please join our [Discord server](https://discord.gg/26wMygt)**

* **I'm submitting a ...**
[x] bug report
[ ] feature request
[ ] question about the decisions made in the repository

* **Action taken** (what you did)
Started live trading for BTC/XRP on Bitfinex. Bot starts and buys XRP. Still, the bot will try to buy XRP at next Signal and since the amount then is too low, will get an error. Bot then crashes because of amount too small. Same happens when orders are only fulfilled partially.

* **Expected result** (what you hoped would happen)
Stable trading of XRP/BTC with RSI strat.

* **Actual result** (unexpected outcome)
```
2018-01-19 00:20:19 (INFO):     Attempting to BUY 29.55338466008135 XRP at Bitfinex price: 0.0001375
2018-01-19 00:20:38 (DEBUG):    Requested XRP/BTC trade data from Bitfinex ...
2018-01-19 00:20:38 (DEBUG):    Processing 3 new trades. From 2018-01-18 23:20:26 UTC to 2018-01-18 23:20:27 UTC. (a few seconds)
2018-01-19 00:20:58 (DEBUG):    Requested XRP/BTC trade data from Bitfinex ...
2018-01-19 00:20:59 (DEBUG):    Processing 8 new trades. From 2018-01-18 23:20:40 UTC to 2018-01-18 23:20:56 UTC. (a few seconds)
2018-01-19 00:21:18 (DEBUG):    Requested XRP/BTC trade data from Bitfinex ...
2018-01-19 00:21:18 (DEBUG):    Processing 5 new trades. From 2018-01-18 23:21:13 UTC to 2018-01-18 23:21:16 UTC. (a few seconds)
2018-01-19 00:21:19 (INFO):     BUY order was not (fully) filled, cancelling and creating new order
2018-01-19 00:21:21 (INFO):     Attempting to BUY 0.038508867173362285 XRP at Bitfinex price: 0.00013807
2018-01-19 00:21:21 (ERROR):    [bitfinex.js] (submitOrder) returned an irrecoverable error: Invalid order: minimum size for XRP/BTC is 10.0


Gekko encountered an error and can't continue

Error:

[bitfinex.js] Invalid order: minimum size for XRP/BTC is 10.0



Meta debug info:

Gekko version: v0.5.12
Nodejs version: v8.9.3
```

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc)

**Comment by cmroche** [comment]
This was likely fixed in PR #1550 which was merged into the develop branch today, please sync the develop branch and confirm if that fixes your issue.

**Comment by stefano-neri** [comment]
actually i manually set "minimalOrder" in bitfinex.js for the XRP/BTC pair:
`
Trader.getCapabilities = function () {
  return {
    name: 'Bitfinex',
    slug: 'bitfinex',
    currencies: ['USD', 'BTC', 'ETH'],
    assets: ['BTC', 'LTC', 'ETH', 'SAN', 'IOT', 'BCH', 'OMG', 'XMR', 'DSH', 'ZEC', 'EOS', 'ETC', 'XRP', 'NEO', 'ETP'],
    markets: [
      
        //Tradeable Pairs to USD
        { pair: ['USD', 'BTC'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['USD', 'BCH'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['USD', 'IOT'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['USD', 'OMG'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['USD', 'EOS'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['USD', 'DSH'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['USD', 'SAN'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['USD', 'ETH'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['USD', 'LTC'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['USD', 'ZEC'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['USD', 'XMR'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['USD', 'ETC'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['USD', 'XRP'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['USD', 'NEO'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['USD', 'ETP'], minimalOrder: { amount: 0.01, unit: 'asset' } },
      
        //Tradeable Pairs to BTC
        { pair: ['BTC', 'ETH'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['BTC', 'BCH'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['BTC', 'IOT'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['BTC', 'OMG'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['BTC', 'DSH'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['BTC', 'ZEC'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['BTC', 'XMR'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['BTC', 'LTC'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['BTC', 'SAN'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['BTC', 'EOS'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['BTC', 'ETC'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['BTC', 'XRP'], minimalOrder: { amount: 10.0, unit: 'asset' } },
        { pair: ['BTC', 'NEO'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['BTC', 'ETP'], minimalOrder: { amount: 0.01, unit: 'asset' } },
      
        //Tradeable Pairs to ETH
        { pair: ['ETH', 'BCH'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['ETH', 'IOT'], minimalOrder: { amount: 6.00, unit: 'asset' } },
        { pair: ['ETH', 'OMG'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['ETH', 'SAN'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['ETH', 'EOS'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['ETH', 'NEO'], minimalOrder: { amount: 0.01, unit: 'asset' } },
        { pair: ['ETH', 'ETP'], minimalOrder: { amount: 0.01, unit: 'asset' } },
      
    ],
    requires: ['key', 'secret'],
    tid: 'tid',
    providesFullHistory: true,
    tradable: true
  };
}`
and it is running quite well since the change...
